packages/bitcore-client/bin/wallet-send

```diff
@@ -29,12 +29,12 @@ const main = async () => {
   try {
     wallet = await Wallet.loadWallet({ name, path });
     const chain = wallet.chain;
-    const { decimals } = currencies.data.find(element => element.code === chain);
+    const { decimals } = currencies.data.find(element => element.code === (token || chain));
```

Comment:
can you do 
```
const currency = token  || chain;
...
element.code === currency
```

Other than that, looks good.

Reply:
done